packages/collision/index.ts

```diff
@@ -1,233 +1,648 @@
-import compose, { ComposeProperty, Composer, ComposerParams, Descriptor, PropertyMap, Stamp } from '@stamp/compose';
-import { assign } from '@stamp/core';
-import { isArray, isObject, isStamp } from '@stamp/is';
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable prettier/prettier */
```

Comment:
@sammys can you please re-enable `prettier/prettier`? Many lines are flagged as changed just because of the missing trailing semi-colon.

Reply:
Done in next commit.